packages/client/hmi-client/src/components/model/model-parts/tera-model-part-entry.vue

```diff
@@ -1,5 +1,12 @@
 <template>
 	<section class="flex flex-column">
+		<div v-if="modelErrors.length > 0">
+			<ul>
+				<li v-for="err of modelErrors" :key="err.id">
+					<span :class="err.severity">{{ err.content }}</span>
+				</li>
+			</ul>
+		</div>
```

Comment:
We are constitently use `isEmpty` to do those conditional test.
```suggestion
			<ul v-if="!isEmtpy(modelErrors)">
				<li v-for="err of modelErrors" :key="err.id">
					<span :class="err.severity">{{ err.content }}</span>
				</li>
			</ul>
```

Reply:
done